packages/back-office-subscribers/package.json

```diff
@@ -9,7 +9,10 @@
 	},
 	"dependencies": {
 		"axios": "^1.4.0",
-		"lodash.pick": "^4.4.0"
+		"express": "4.18.2",
+		"lodash.pick": "^4.4.0",
+		"multi-semantic-release": "^2.13.0",
+		"sequelize": "6.33.0"
```

Comment:
don't think some of these dependencies are needed in `back-office-subscribers` - definitely not using `express`, or `sequelize` in this package. I presume `multi-semantic-release` should be in the root `package.json` too? might even be a devDependency?

Reply:
yeah i figured it was some automatic thing. Might be best to remove manually. Should only need `lodash.pick` out of those for this package